mautic/mautic
# Remove deprecated %kernel.project_dir% from config.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x ]
| Automated tests included?              | [ x ] 



#### Description:



#### Steps to test this PR:
Schedule contacts to export. Run `mautic:contacts:scheduled_export` command.

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Schedule contacts to export. Run `mautic:contacts:scheduled_export` command.
